src/bundle/Resources/views/themes/admin/ui/search/form.html.twig

```diff
@@ -97,6 +98,24 @@
             </svg>
             <ul class="ez-filters__user-list ez-filters__user-list--hidden"></ul>
         </div>
+        <div class="ez-date-range ez-date-range--select-modified-range {% if form.last_modified_select.vars.data == "custom_range" %}ez-date-range--visible{% endif %}" data-period-selector="#search_last_modified_date_interval" data-end-selector="#search_last_modified_end_date">
```

Comment:
I think this should be inside the div `ez-filters__item ez-filters__item--modified` because it's related to this select.

Also, I would not introduce a new component so this should be named `ez-filters__range-wrapper`